searx/engines/base.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+"""
+ BASE (Scholar publications)
+
+ @website     https://base-search.net
+ @provide-api yes with authorization (https://api.base-search.net/)
+
+ @using-api   yes
+ @results     XML
+ @stable      ?
+ @parse       ?
```

Comment:
Values of parse must be the keys of the results dict. In this case it is url, title, publishedDate, content.